packages/core-data/README.md

```diff
@@ -976,6 +976,47 @@ The following set of react hooks available to import from the `@wordpress/core-d
 
 <!-- START TOKEN(Autogenerated hooks|src/hooks/index.ts) -->
 
+### useEntityBlockEditor
```

Comment:
Why is the EntityProvider not present in the docs. AFAIK, it's exported in the file as well?

Reply:
It exported as the default but because it's a component I didn't move it's export into hooks.